Assignment2/alvaroalvi-fromJSONtoJSON-LD.jsonld

```diff
@@ -1,27 +1,38 @@
 {
-  "@context": "http://schema.org/",
- "@id": "http://schema.org/name",
-  "@type": "Person",
-  "name": "Alvaro Garcia Garcia",
-  "hometown":"Madrid",
-  "age": "23",
-  "Almamater":"ETSIINF",
-  "Parentsname":"Santi y Macarena",
-  "subjects": [
-	{
-		"name":"Sistemas Operativos",
-		"teacher":"Rosales",
-		"description":"conocer el funcionamiento interno de un ordenador"
-		},
-		{
-		"name":"Web Semantica",
-		"teacher":"Oscar",
-		"description":"serializaciones en paginas web"
-		},
-		{
-		"name":"Nanotecnologia",
-		"teacher":"Estibaliz",
-		"description":"estudio de el mundo nano y sus caracteristicas"
-		}
-	]
+	"@context":"https://schema.org",
+	"@type":"Person",

```

Comment:
You have to provide an id. 
The problem with your previous id definition was you also using http://schema.org/name for your annotate your name. Please, provide a different id as a way to identify the resource.